src/scss/config/scale/_theme-sizes.scss

```diff
@@ -1,4 +1,11 @@
+@use 'sass:math';
+@use 'page-sizes';
```

Comment:
these aren't necessary with what is currently in the file. 
```suggestion
```

Reply:
whoops. removed in https://github.com/oddbird/oddleventy/pull/306/commits/088d051b4dd722f80ad7747508b815562102895c